src/aws-credentials/provider.cr

```diff
@@ -44,14 +44,14 @@ module Aws::Credentials
     end
 
     private def resolve_credentials : Credentials
-      @providers.find { |p|
-        p.credentials? ? true : false
+      @providers.find { |provider_|
```

Comment:
[Q] why ends with underscore?

Reply:
Ameba forbids short names (e.g. `p`, `pp`), and I was out of creativity for a better name.